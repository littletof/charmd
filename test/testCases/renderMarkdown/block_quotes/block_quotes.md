> Blockquotes can also be nested...
>> ...by using additional greater-than signs right next to each other...
> > > ...or with spaces between arrows.

> Some quote
and more

>> Another quote

> Some quote
> with **bold** and `block`
> > and Another quote
>> >> #### and some more quote