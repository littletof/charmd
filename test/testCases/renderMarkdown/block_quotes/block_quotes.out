â”ƒ [90m[3mBlockquotes can also be nested...[23m[39m
â”ƒ [90m[3mâ”ƒ [90m[3m...by using additional greater-than signs right next to each other...[3m[90m[23m[39m
â”ƒ [90m[3mâ”ƒ [90m[3mâ”ƒ [90m[3m...or with spaces between arrows.[3m[90m[3m[90m[23m[39m

â”ƒ [90m[3mSome quote[23m[39m
â”ƒ [90m[3mand more[23m[39m

â”ƒ [90m[3mâ”ƒ [90m[3mAnother quote[3m[90m[23m[39m

â”ƒ [90m[3mSome quote[23m[39m
â”ƒ [90m[3mwith [1mbold[22m and [7m block [27m[23m[39m
â”ƒ [90m[3mâ”ƒ [90m[3mand Another quote[3m[90m[23m[39m
â”ƒ [90m[3mâ”ƒ [90m[3mâ”ƒ [90m[3mâ”ƒ [90m[3m[35m[1m#### and some more quote[22m[90m[3m[90m[3m[90m[3m[90m[23m[39m
